tools/Docs/GenerateDotNetCsv.ps1

```diff
@@ -0,0 +1,22 @@
+Param(
+    [Parameter(Mandatory = $true)]
+    [string]$FeedPsd1FullPath
+)
+
+$feedDir = (Get-Item $FeedPsd1FullPath).Directory
+$feedName = (Get-Item $FeedPsd1FullPath).Name
+Import-LocalizedData -BindingVariable ModuleMetadata -BaseDirectory $feedDir -FileName $feedName
+$modules = $ModuleMetadata.RequiredModules
+
+$dotnetCsv = New-Item -Path "$PSScriptRoot\" -Name "az-ps-latest.csv" -ItemType "file" -Force
+$dotnetCsvContent = ""
+for ($index = 0; $index -lt $modules.Count; $index++){
+    $moduleName = $modules[$index].ModuleName
+    $moduleVersion = if([string]::IsNullOrEmpty($modules[$index].RequiredVersion)){ $modules[$index].ModuleVersion } else{ $modules[$index].RequiredVersion } 
```

Comment:
I believe this script is always run in Powershell 7.0+ , so we can use [Ternary operator ?  : ](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7#ternary-operator--if-true--if-false) to simplify the script
```suggestion
    $moduleVersion = [string]::IsNullOrEmpty($modules[$index].RequiredVersion)) ? $modules[$index].ModuleVersion : $modules[$index].RequiredVersion
```

Reply:
Just tried in Pwsh7.1.3, look like it has not supported ternary operator. For compatibility, I prefer to keep the old style.
![image](https://user-images.githubusercontent.com/19869716/126939899-35494008-aa62-416e-b3c7-b040f593203d.png)

Reply:
Daily build pipeline still uses 6.2.4 to bump version. This script won't work in this case
![image](https://user-images.githubusercontent.com/19869716/126942344-54bcee39-f21e-466c-a289-a8bb62cdc62f.png)